File: Pacman-Game-OOP-master 2/EASendMail/Samples_NetDesktop/VB/Mass/AppProject1/FormRecipient.vb
Public Class FormRecipient
    Public Sub New()
        MyBase.New()
        InitializeComponent()
    End Sub

    Private Sub ButtonCancel_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles ButtonCancel.Click
        Return
    End Sub

    Private Sub ButtonOK_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles ButtonOK.Click
        TextBoxName.Text = TextBoxName.Text.Trim()
        TextBoxAddress.Text = TextBoxAddress.Text.Trim()

        If TextBoxAddress.Text.Length = 0 Then
            MessageBox.Show(Me, "Please input a valid email address!")
            TextBoxAddress.Focus()
            Return
        End If

        ButtonOK.DialogResult = DialogResult.OK
        Me.DialogResult = DialogResult.OK
        Me.Close()
    End Sub
End Class

File: Pacman-Game-OOP-master 2/EASendMail/Samples_NetDesktop/C#/Mass/AppProject1/FormRecipient.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace AppProject1
{
    public partial class FormRecipient : Form
    {
        public FormRecipient()
        {
            InitializeComponent();
        }

        private void ButtonCancel_Click(object sender, System.EventArgs e)
        {
            return;
        }

        private void ButtonOK_Click(object sender, System.EventArgs e)
        {
            TextBoxName.Text = TextBoxName.Text.Trim();
            TextBoxAddress.Text = TextBoxAddress.Text.Trim();
            if (TextBoxAddress.Text.Length == 0)
            {
                MessageBox.Show(this, "Please input a valid email address!");
                TextBoxAddress.Focus();
                return;
            }

            ButtonOK.DialogResult = DialogResult.OK;
            this.DialogResult = DialogResult.OK;
            this.Close();
        }
    }
}
